### DISCUSSION TITLE
Render HTML showing (but not executing) python code

### DISCUSSION THREAD

**Discussion by eriskar** [quarto-dev/quarto-cli#4161] [Q&A] [answered]
Hey, all

I am currently trying to render a HTML with Quarto to show some python code blocks to students. I've done the same in the past with R code and I have avoided the code from running when rendering the HTML with ```{r, eval=F, echo=T} ´´´, but it does not seem to work with ´´´{python}.  

I have also added in YAML:

execute:
  echo: false

Any idea what I can do?

Thanks!

**Comment by cscheid** [comment, marked as answer]
https://quarto.org/docs/computations/execution-options.html

````
---
title: hi
---

```{python}
#| eval: false
import this_will_not_run
```
````

**Reply by eriskar** [reply]
Thank you so much, Carlo. That made the trick along with |# echo: true.

You're an everyday hero!
